src/API/src/db/bionomics/entities/bionomics.entity.ts

```diff
@@ -109,6 +109,10 @@ export class Bionomics extends BaseEntity {
   @Field({ nullable: true })
   data_checked_by: string;
 
+  @Column('varchar', { length: 50, nullable: false })
```

Comment:
Can we remove the `length` bit? We don't really need these constraints any more, and they are not checked by the validator anyway. And I think this should be nullable. You'll need to change the migration for these changes too.